### ISSUE TITLE
Standalone server does not start Firefox with the right profile

### ISSUE THREAD

**Issue by lmtierney** [SeleniumHQ/selenium#3331]
This is happening on the current master branch but not with the previous official release of Standalone. Firefox is starting, but not with the correct profile.

To reproduce, build the latest standalone from master and run the following:

    from selenium import webdriver
    import pdb
    profile = webdriver.FirefoxProfile()
    profile.set_preference('browser.startup.homepage', 'http://www.google.com')
    profile.set_preference('browser.startup.page', 1)

    driver = webdriver.Remote(browser_profile=profile, desired_capabilities=webdriver.DesiredCapabilities.FIREFOX)

    pdb.set_trace()
    driver.quit()

It opens with the correct profile without using remote:

    from selenium import webdriver
    import pdb
    profile = webdriver.FirefoxProfile()
    profile.set_preference('browser.startup.homepage', 'http://www.google.com')
    profile.set_preference('browser.startup.page', 1)

    driver = webdriver.Firefox(firefox_profile=profile)

    pdb.set_trace()
    driver.quit()

The commit that broke it was 7264b8a, I reverted to the commit before it and confirmed that it was working correctly.

When running the server with debug, this is the message after attempting to start a session with the profile:

```
1483672590133	mozprofile::profile	INFO	Using profile path /var/folders/lp/kxd1t1m9113fpq4fv8r2gyrw0000gq/T/rust_mozprofile.JzT06kdHJPtF
21:16:30.134 DEBUG - http-outgoing-0 << "HTTP/1.1 500 Internal Server Error[\r][\n]"
21:16:30.134 DEBUG - http-outgoing-0 << "Connection: close[\r][\n]"
21:16:30.134 DEBUG - http-outgoing-0 << "Content-Type: application/json[\r][\n]"
21:16:30.134 DEBUG - http-outgoing-0 << "Content-Length: 61[\r][\n]"
21:16:30.134 DEBUG - http-outgoing-0 << "Date: Fri, 06 Jan 2017 03:16:30 GMT[\r][\n]"
21:16:30.134 DEBUG - http-outgoing-0 << "[\r][\n]"
21:16:30.135 DEBUG - http-outgoing-0 << "{"error":"unknown error","message":"Failed to unzip profile"}"
```

**Comment by lmtierney** [comment]
I was able to fix it with this commit 9b41dd5, however a more recent commit (9b41dd5) is now sending the profile in `moz:profileOptions` as `firefox_profile` instead of the required `profile`

**Comment by lmtierney** [comment]
Closed via 4e076ee and 09b102f
